.github/workflows/ci.yml

```
@@ -327,3 +327,16 @@ jobs:
           prerelease: false
           files: |
             package-*/*
+            
+  winget-publish:
+    needs: release
+    if: (runner.os == 'Windows') && contains(github.ref, 'tags/v')
```

```suggestion
    if: contains(github.ref, 'tags/v')
```

I realised after thinking about this that `(runner.os == 'Windows')` is redundant because of `runs-on: windows-latest`